src/http/server/response.cr

```diff
@@ -40,6 +40,26 @@ class HTTP::Server
       output.response = self
     end
 
+    def remote_address
+      case @io
+      when IPSocket, UNIXSocket
+        io = @io
+      when OpenSSL::SSL::Socket
+        io = @io.as(OpenSSL::SSL::Socket).io
+      else
+        raise "Unsupported IO"
+      end
+
+      case io
+      when IPSocket, UNIXSocket
+        remote_address = io.remote_address
+      else
+        raise "Unsupported socket"
+      end
+
+      remote_address
```

Comment:
Actually this variable can be removed too only leaving `case`.

Reply:
Would it still be worth leaving for readability?

Reply:
I'm not sure, it (readability) is pretty subjective but perhaps... anyone else would like to chime in?